keps/prod-readiness/README.md

```diff
@@ -20,3 +20,14 @@ Full documentation on the production readiness review process can be found on
 the SIG Architecture page [here][prod-readiness].
 
 [prod-readiness]: https://git.k8s.io/community/sig-architecture/production-readiness.md
+
+## How do I find a PRR reviewer for my KEP?
+
+The KEP template production readiness questionnaire should be filled out by the KEP authors, and this will be reviewed by the SIG leads. Once the leads are satisfied with both the overall KEP (i.e., it is ready to move to `implementable` state) and the PRR answers, the authors may request PRR approval.
+Make sure the `kep.yaml` and README are ready for review and have the correct sections filled out. See the [KEP template](https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template). 
+
+1. Make sure the Enhancement KEP is labeled `lead-opted-in` before PRR Freeze. This is required so that the Enhancements release team and PRR team are aware the KEP is targeting the release.
```

Comment:
```suggestion
1. Before the PRR Freeze, ensure that the GitHub issue for the Enhancement KEP is labeled as `lead-opted-in`. This is required so that both the Enhancements release team and the PRR team are aware that the KEP is targeting the upcoming release milestone.
```